### PR TITLE
CLI content commands - output created entity ids

### PR DESCRIPTION
**Pull request by Lezek123**
Affected commands:
- `content:createVideo`
- `content:createChannel`
- `content:createChannelCategory`
- `content:createVideoCategory`

### CODE REVIEW

**Review by bwhm — APPROVED** [review]
Haven't reviewed the code, but can confirm that all the four commands works as intended!

File: cli/src/base/ApiCommandBase.ts [hunk 1]
```diff
@@ -424,12 +425,16 @@ export default abstract class ApiCommandBase extends StateAwareCommandBase {
...
     params: CodecArg[],
     warnOnly = false
-  ): Promise<boolean> {
+  ): Promise<SubmittableResult | false> {
     this.log(chalk.magentaBright(`\nSending ${module}.${method} extrinsic...`))
     const tx = await this.getOriginalApi().tx[module][method](...params)
     return await this.sendAndFollowTx(account, tx, warnOnly)
   }
 
+  public findEvent(result: SubmittableResult, section: string, method: string): Event | undefined {
```
Comment (mnaamani) [inline cli/src/base/ApiCommandBase.ts:439]: There is actually a `.findRecord(section, method)` method on the `SubmittableResult` type that i've used here for example https://github.com/Joystream/joystream/blob/master/storage-node/packages/runtime-api/index.js#L336

File: cli/src/base/ApiCommandBase.ts [hunk 2]
```diff
@@ -401,11 +402,11 @@ export default abstract class ApiCommandBase extends StateAwareCommandBase {
     account: KeyringPair,
     tx: SubmittableExtrinsic<'promise'>,
     warnOnly = false // If specified - only warning will be displayed in case of failure (instead of error beeing thrown)
-  ): Promise<boolean> {
+  ): Promise<SubmittableResult | false> {
```
Comment (mnaamani) [inline cli/src/base/ApiCommandBase.ts:405]: I probably would have gone with `<SubmittableResult | null>` which would make a bit more sense but this works too.

**Review by mnaamani — APPROVED** [review]
Looks good, only minor suggestion but not critical.

**Pull request merged**